x/emissions/module/rewards/scores.go

```diff
@@ -15,7 +17,17 @@ import (
 
 // GenerateReputerScores calculates and persists scores for reputers based on their reported losses.
 func GenerateReputerScores(ctx sdk.Context, keeper keeper.Keeper, topicId uint64, block int64, reportedLosses types.ReputerValueBundles) ([]types.Score, error) {
-	// Get reputers data
+	// Fetch all workers for the topic to determine expected report coverage
+	allWorkers, err := keeper.GetTopicWorkers(ctx, topicId)
```

Comment:
Note I'm probs gonna be applying this ticket here to bound our computation:
https://linear.app/upshot/issue/ORA-1060/skim-only-the-top-n-r-n-w-reputers-and-workers-respectively-for-each

to here or to the tx where the reputer data is saved in the first place.